RecoTracker/IterativeTracking/python/PixelLessStep_cff.py

```diff
@@ -206,6 +206,25 @@
         )
     )
 )
+
+from Configuration.ProcessModifiers.approximatedClusters_cff import approximatedClusters
+approximatedClusters.toModify(pixelLessStepSeeds,
```

Comment:
was this just meant to remove `SeedComparitorPSet.comparitors[1]` ? please simplify